src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -174,6 +174,22 @@ public void setName(Name name) {
             return Optional.ofNullable(name);
         }
 
+        public void setAddress(Address address) {
+            this.address = address;
+        }
+
+        public Optional<Address> getAddress() {
+            return Optional.ofNullable(address);
+        }
+
+        public void setSeller(Name seller) {
+            this.seller = seller;
+        }
+
+        public Optional<Name> getSeller() {
+            return Optional.ofNullable(seller);
+        }
```

Comment:
Should this be `setSellerName` and `getSellerName` respectively? Otherwise it may seem like this is referring to the `Person` seller and not just the seller's `Name`

Reply:
Yup, will be prefixing seller fields with seller as with lizchow's suggestions.